docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-19_remove_regridded.txt

```diff
@@ -0,0 +1 @@
+* The deprecated method ```iris.cube.Cube.regridded``` has been removed from the module :mod:`iris.cube`.
```

Comment:
@corinnebosley `iris.cube.Cube.add_history` and `iris.cube.Cube.assert_valid` have also been removed (but before this lot of work) - see https://github.com/SciTools/iris/compare/v1.13.x...master?expand=1#diff-d1f30240d0d146f09f58577e718ba6a6L3122 and https://github.com/SciTools/iris/compare/v1.13.x...master?expand=1#diff-d1f30240d0d146f09f58577e718ba6a6L2139, respectively.

Similarly, the deprecated kwargs `coord` and `name` have been removed from `iris.cube.Cube.coord` and `iris.cube.Cube.coords` - see from https://github.com/SciTools/iris/compare/v1.13.x...master?expand=1#diff-d1f30240d0d146f09f58577e718ba6a6L1242.